generators/client/templates/angular/package.json.ejs

```diff
@@ -180,6 +181,11 @@
     "build": "<%= clientPackageManager %> run webpack:prod",
     "test": "<%= clientPackageManager %> run lint && jest --coverage --logHeapUsage -w=2 --config src/test/javascript/jest.conf.js",
     "test:watch": "<%= clientPackageManager %> run test <%= optionsForwarder %>--watch",
+    <%_ if (buildTool === 'gradle') { _%>
+    "typedoc": "typedoc --out build/docs/html5/typedocs src/main/webapp/app",
+    <%_ } else { _%>
```

Comment:
Here you assume maven is available, but this is not always true since you can generate a client only application, with the `--skip-server` option

Reply:
Sorry, forget my last comment, I read it too quickly...